pulumi/folio/vpc.ts

```diff
@@ -76,6 +77,12 @@ export module deploy {
                 toPort: 9000,
                 protocol: "tcp",
                 cidrBlocks: ["0.0.0.0/0"]
+            }, {
+                description: "Allow inbound traffic on 5432 for postgres",
+                fromPort: 5432,
+                toPort: 5432,
+                protocol: "tcp",
+                cidrBlocks: ["0.0.0.0/0"]
```

Comment:
Does this allow all inbound traffic? That seems like something we really don't want

Reply:
I believe the right way to handle this is to set the `cidrBlocks` parameter for the `5432` ingress to be the CIDR block of the VPC. We should be able to do this because the VPC is created before the security group, so we should be able to get that CIDR range after creating the VPC and pass it into the creation of the security group.

Or we can specify the CIDR range within when creating the VPC and then reuse that when creating the security group.

Reply:
Also related to the `0.0.0.0/0` ingress for `5432`, the `rds.SubnetGroup` is limited to the VPC `privateSubnetIds`, and so far in my experience for the cluster to get opened up to the outside world you have punch open a hole with a `LoadBalancer` service. 

Either way the CIDR block here shouldn't `0.0.0.0/0' but instead should be the same as what we designate when we create the VPC. I'll make and test out that change.

For reference see: https://www.pulumi.com/docs/guides/crosswalk/aws/vpc/#configuring-cidr-blocks-for-a-vpc. 

Right now we're just taking the default CIDR which starts with `10.0` (all pods start with this IP for example), which is probably what I'll start with. But it seems like you can set any "mask" you want.

Reply:
See the latest commit. I'm now limiting the security group CIDR to `10.0.0.0/16` for the pg port.